src/client/kotlin/tech/thatgravyboat/skyblockapi/api/remote/repo/RepoSlayerData.kt

```diff
@@ -0,0 +1,43 @@
+package tech.thatgravyboat.skyblockapi.api.remote.repo
+
+import com.google.gson.JsonObject
+import com.mojang.serialization.Codec
+import tech.thatgravyboat.skyblockapi.api.SkyBlockAPI
+import tech.thatgravyboat.skyblockapi.api.area.slayer.SlayerType
+import tech.thatgravyboat.skyblockapi.generated.KCodec
+import tech.thatgravyboat.skyblockapi.kcodec.GenerateCodec
+import tech.thatgravyboat.skyblockapi.modules.FieldName
+import tech.thatgravyboat.skyblockapi.modules.Module
+import tech.thatgravyboat.skyblockapi.utils.codecs.CodecUtils
+import tech.thatgravyboat.skyblockapi.utils.json.Json.readJson
+import tech.thatgravyboat.skyblockapi.utils.json.Json.toDataOrThrow
+import java.nio.file.Files
+
+@Module
+object RepoSlayerData {
+
+    private val CODEC: Codec<MutableMap<SlayerType, RepoSlayerData>> =
+        CodecUtils.map(KCodec.getCodec<SlayerType>(), RepoSlayerData.CODEC)
```

Comment:
since it doesnt actually need to be mutable use Codec.unboundMap instead and inline this in the toDataOrThrow call